src/ui/components/UploadScreen/Privacy.tsx

```diff
@@ -86,7 +86,7 @@ export function Privacy() {
   const { operations } = recording!;
 
   return (
-    <div className="flex w-full h-full rounded-xl shadow-xl text-lg relative p-8 bg-jellyfish overflow-hidden">
+    <div className="flex w-full h-full rounded-xl shadow-xl text-md relative p-8 bg-white overflow-hidden">
```

Comment:
this should be a conditional so it is a jelly fish in the upload screen and white in devtools

Reply:
Aw crap. Thanks.